### ISSUE TITLE
train with my own datasets, the results is unstable

### ISSUE THREAD

**Issue by zener90818** [cherubicXN/hawp#42]
Hi,
Thanks for your sharing.
I train with my own datasets. The results are as below：
Two consecutive images are extracted from a video stream. the detected line is unstable,  What reason is this? 
Thanks a lot.

![Uploading 微信截图_20230217153519.png…]()
![微信截图_20230217153439](https://user-images.githubusercontent.com/38679014/219580072-d4d649eb-80f2-4eb6-83e4-97d348e9e99d.png)

**Comment by cherubicXN** [comment]
Yes, it is indeed a problem. Another example can be found in a [Youtube video](https://www.youtube.com/watch?v=z3cWRb5AVPc). The main reason for this problem is that we did not consider the temporal information in both our designs and training.

**Comment by yizhidecainiao** [comment]
> Hi, Thanks for your sharing. I train with my own datasets. The results are as below： Two consecutive images are extracted from a video stream. the detected line is unstable, What reason is this? Thanks a lot.
> 
> ![微信截图_20230217153439](https://user-images.githubusercontent.com/38679014/219581069-eed47d82-4fef-4dbe-8466-01a981d9f08f.png) ![微信截图_20230217153519](https://user-images.githubusercontent.com/38679014/219581091-a196c940-d12a-41c0-aff0-f057baa080b5.png)

Hi, how do you train your own data, can you share your configuration file and the training commands, thanks a lot!
